applications/luci-app-babeld/htdocs/luci-static/resources/babeld.js

```diff
@@ -0,0 +1,171 @@
+function renderTableXRoutes(ubus_data, target_id) {
+	var data = JSON.parse(ubus_data);
+	for (var protocol in data) {
+		var target = document.getElementById(target_id);
+
+		var title = document.createElement('h3');
+		title.appendChild(document.createTextNode('X-Routes ' + protocol));
+		target.appendChild(title);
+
+		var table = document.createElement('table');
+		table.setAttribute('class', 'table');
+		table.setAttribute('id', 'babel_overview_xroutes_' + protocol);
+
+		var headerRow = document.createElement('tr');
+		headerRow.setAttribute('class', 'tr table-titles');
+		var headerContent = '<th class="th" style="font-weight: 700;">' + protocol + ' Prefix</th>\
```

Comment:
This could be a potential XSS vector. Consider using `'%h'.format(protocol)` for HTML escaping

Reply:
Okay! I'll fix this, Actually I kept it almost same as the old code. Thanks